planck-cljs/test/planck/repl_test.cljs

```diff
@@ -50,9 +50,9 @@
 (defspec add-drop-macros-suffix-test
   (prop/for-all [ns-sym gen/symbol]
     (= ns-sym (-> ns-sym
-                ^:private-var-access-nowarn repl/add-macros-suffix
+                (#'repl/add-macros-suffix)
```

Comment:
This is really a more elementary question but what is this line doing? I read it as the `` #` `` syntax getting the variable represented by `repl/add-macros-suffix` but then that passes the result of the previous line to it as an argument? Assuming I'm reading that correctly, what does it mean to call a function with a var?

Reply:
Oh, is this a workaround to allow a private function to be called from outside the namespace?

Reply:
Yes. IIRC there is a special evaluation rule that says that a if a var is in operator position it will be dereference to turn it into its function value.

The need for the parens is interesting, without  them the code looks like (ignoring the remainder of the chain):

```clojure
(-> ns-sym
  (var repl/add-macros-suffix))
```

which is not what you want because this expands to:

```
(var ns-sym repl/add-macros-suffix)
```

Two ways to fix that are to either use `->>` or to add parens.

Reply:
Got it! Thanks for the explanation!